src/NSwag.Commands/Commands/Generation/AspNetCore/AspNetCoreToOpenApiCommand.cs

```diff
@@ -61,6 +61,9 @@ public class AspNetCoreToSwaggerCommand : OpenApiGeneratorCommandBase<AspNetCore
         [Argument(Name = nameof(NoBuild), IsRequired = false, Description = "Don't build the project. Only use this when the build is up-to-date.")]
         public bool NoBuild { get; set; }
 
+        [Argument(Name = nameof(OutputPath), IsRequired = false, Description = "Path to the MSBUILD output directory")]
+        public string OutputPath { get; set; }
```

Comment:
Not sure whether "OutputPath" is a good name here as I'd expect the OpenAPI.json output path with that... 
Maybe "BuildOutputPath" (msbuild is legacy :-)?

Reply:
OutputPath is indeed a bit confusing. I have renamed it to MSBuildOutputPath